examples/04-advanced/14-incremental_evaluation.py

```diff
@@ -0,0 +1,34 @@
+"""
+.. _ref_incremental_evaluation:
+"""
+
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+
+def scop_from_ds(ds):
+    time_freq_provider = dpf.operators.metadata.time_freq_provider(data_sources=ds)
+    tf_support = time_freq_provider.get_output(output_type=dpf.types.time_freq_support)
+    return dpf.time_freq_scoping_factory.scoping_on_all_time_freqs(tf_support)
+
+path = examples.download_transient_result()
+ds = dpf.DataSources(path)
+scoping = scop_from_ds(ds)
+
+
+result_op = dpf.operators.result.displacement()
```

Comment:
We try to not use the syntax result_op.connect(4, ds) in examples, do you think that you could instead do: 
```suggestion
result_op = dpf.operators.result.displacement(data_sources=ds)
```